src/main/java/com/thebois/models/beings/AbstractBeing.java

```diff
@@ -37,12 +38,25 @@
      * How many kilograms a being can carry.
      */
     private static final float MAX_CARRYING_CAPACITY = 100f;
+    private static final float MAX_HEALTH = 100f;
+    private static final float MAX_HUNGER = 100f;
+    /**
+     * How much hunger the pawn should lose per second.
+     */
+    private static final float HUNGER_FACTOR = 1f;
+    /**
+     * How much health the pawn should gain or lose per second.
+     */
+    private static final float HEALTH_FACTOR = 1f;
     private final IPathFinder pathFinder;
     private final IInventory inventory = new Inventory(MAX_CARRYING_CAPACITY);
     private Stack<Position> path;
     private Position position;
     private AbstractRole role;
     private final IStructureFinder finder;
+    private float currentHunger;
+    private float currentHealth;
```

Comment:
style

Imo it's unnecessary to prefix them with "current" since what the hunger and health of a being are at are always their "current" values.